packtools/sps/validation/aff.py

```diff
@@ -418,6 +428,66 @@ def validate_comparison(self, main_aff: dict):
                 data=items,
             )
 
+    def validate_orgname_components(self):
+        if not self.original:
+            return
+
+        component_not_found = [
+            key for key, value in self.original_components.items()
+            if value and not any(word in self.original for word in value.split())
+        ]
```

Comment:
@Rossi-Luciano Adicione à mensagem além do key o valor.
Por exemplo:

Troque
` and add missing components: {component_not_found}.` 
por `add 'Brasil (country), São Paulo (city) in <institution content-type="original">`

Reply:
OK, @robertatakenaka